airbyte-integrations/connectors/source-notion/unit_tests/test_streams.py

```diff
@@ -76,3 +78,56 @@ def test_backoff_time(patch_base_class):
     stream = NotionStream(config=MagicMock())
     expected_backoff_time = None
     assert stream.backoff_time(response_mock) == expected_backoff_time
+
+
+def test_users_request_params(patch_base_class):
+    stream = Users(config=MagicMock())
+
+    # No next_page_token. First pull
+    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": None}
+    expected_params = {"page_size": 100}
+    assert stream.request_params(**inputs) == expected_params
+
+    # When getting pages after the first pull.
+    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": {"next_cursor": "123"}}
+    expected_params = {"start_cursor": "123", "page_size": 100}
+    assert stream.request_params(**inputs) == expected_params
```

Comment:
nit: I suggest leverage parametrize wherever you can.
```suggestion
@pytest.mark.parametrize("next_page_token, expected_params", [(None, {"page_size": 100}), ({"next_cursor": "123"}, {"start_cursor": "123", "page_size": 100})])
def test_users_request_params(patch_base_class, next_page_token, expected_params):
    stream = Users(config=MagicMock())
    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": next_page_token}
    assert stream.request_params(**inputs) == expected_params
```